facet-testhelpers/src/lib.rs

```diff
@@ -35,6 +35,6 @@ pub fn setup() {
     #[cfg(not(miri))]
     color_backtrace::install();
     let logger = Box::new(SimpleLogger);
-    log::set_boxed_logger(logger).unwrap();
+    _ = log::set_boxed_logger(logger);
```

Comment:
Note: Doesn't really belong in this PR, but I had to add this to get tests to run.